extensions/VisualEditor/modules/ve-mw/init/ve.init.mw.Target.js

```diff
@@ -1348,10 +1352,10 @@ ve.init.mw.Target.prototype.tryWithPreparedCacheKey = function ( doc, options, e
 		}
 		return deflatePromise
 			.then( function () {
-				return new mw.Api().post( data, { contentType: 'multipart/form-data' } );
+				return target.constructor.static.apiRequest( data, { type: 'POST' } );
 			} )
 			.then(
-				function ( response, jqxhr ) {
+				function ( response, status, jqxhr ) {
 					var eventData = {
```

Comment:
We don't need need the status; however, the parameters passed into this function have changed to include the status so we need to represent it here.